### PR TITLE
Login signup style

### PR DESCRIPTION
**Pull request by tsbolty**
### What this PR does (required):
- Completed all styling of login and signup pages

### Screenshots / Videos (required):
image

image

### Any information needed to test this feature (required):
- Login page should load as soon as application renders. To get to sign up page, click the "sign up" button

### Any issues with the current functionality (optional):
- No problems. Fully mobile responsive

### CODE REVIEW

File: client/src/components/AuthHeader/AuthHeader.tsx [hunk 1]
```diff
@@ -2,22 +2,21 @@ import Button from '@material-ui/core/Button';
...
-const AuthHeader = ({ linkTo, asideText, btnText }: Props): JSX.Element => {
+const AuthHeader = ({ linkTo, btnText }: Props): JSX.Element => {
   const classes = useStyles();
 
   return (
     <Box p={1} className={classes.authHeader}>
-      <Typography className={classes.accAside}>{asideText}</Typography>
-      <Link to={linkTo} className={classes.link}>
-        <Button color="inherit" className={classes.accBtn} variant="contained">
+      <Typography color="secondary" className={classes.tattooArt}>TATTOO ART</Typography>
```
Comment (rajivtitus) [inline client/src/components/AuthHeader/AuthHeader.tsx:17]: Some screen readers pickup capital case text as acronyms, consider using the tattooArt class to transform the text to uppercase :)

File: client/src/components/AuthHeader/useStyles.ts [hunk 2]
```diff
@@ -2,11 +2,16 @@ import { makeStyles } from '@material-ui/core/styles';
 
 const useStyles = makeStyles((theme) => ({
   authHeader: {
-    alignSelf: 'flex-end',
-    justifyContent: 'flex-end',
+    // alignSelf: 'flex-end',
```
Comment (rajivtitus) [inline client/src/components/AuthHeader/useStyles.ts:5]: please removed commented styles

File: client/src/pages/Login/LoginForm/useStyles.ts [hunk 3]
```diff
@@ -4,14 +4,23 @@ const useStyles = makeStyles((theme) => ({
   form: {
     width: '100%', // Fix IE 11 issue.
     marginTop: theme.spacing(1),
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'space-around',
+  },
+  textField: {
+    margin: 0,
+    borderWidth: 0.5,
```
Comment (rajivtitus) [inline client/src/pages/Login/LoginForm/useStyles.ts:13]: You can use the CSS Border - Shorthand Property here :)

File: client/src/components/AuthHeader/useStyles.ts [hunk 4]
```diff
@@ -30,6 +35,23 @@ const useStyles = makeStyles((theme) => ({
...
     marginRight: 35,
   },
+  tattooArt: {
+    fontSize: '2em',
+    marginLeft: 20,
+    letterSpacing: '.4em',
+  },
+  loginButton: {
+    color: 'white',
+    borderColor: 'white',
```
Comment (rajivtitus) [inline client/src/components/AuthHeader/useStyles.ts:45]: You can use the CSS Border - Shorthand Property here :)

**Review by rajivtitus — APPROVED** [review]
UI looks good! Just a couple of comments, please address before merging.

Great job adding mobile responsiveness early on 👍🏽

File: client/src/components/AuthHeader/AuthHeader.tsx (hunk 1 above)
Comment (tsbolty) [reply client/src/components/AuthHeader/AuthHeader.tsx:17]: This is fixed.

File: client/src/components/AuthHeader/useStyles.ts (hunk 2 above)
Comment (tsbolty) [reply client/src/components/AuthHeader/useStyles.ts:5]: This is fixed.

File: client/src/components/AuthHeader/useStyles.ts (hunk 4 above)
Comment (tsbolty) [reply client/src/components/AuthHeader/useStyles.ts:45]: There were 3 of these in total. They have all been fixed.

File: client/src/pages/Login/LoginForm/useStyles.ts (hunk 3 above)
Comment (tsbolty) [reply client/src/pages/Login/LoginForm/useStyles.ts:13]: There were 3 of these issues. They are all fixed.

**Comment by tsbolty** [comment]
Completed all styling for login and signup pages. Fully mobile responsive.

**Pull request merged**